InOffice/HeaderSectionCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  HeaderSectionCell.swift
+//  InOffice
+//
+//  Created by Luka Muše on 10.12.2021..
+//
+
+import UIKit
+
+class HeaderSectionCell: UITableViewHeaderFooterView {
+    private var titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 22, weight: .bold)
+        return label
+    }()
+    private var view: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white
+        return view
+    }()
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+        configureView()
+        setupConstraints()
+    }
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    private func configureView() {
+        addSubview(view)
+        view.addSubview(titleLabel)
```

Comment:
you can omit or use viewcontroller view when calling addSubview method, but don't use both